### DISCUSSION TITLE
Low training quality

### DISCUSSION THREAD

**Discussion by leshabirukov** [Ilykuleshov/pytorch-toolz#2] [General]
I have maked adoptation of this tutorial
https://pytorch.org/tutorials/intermediate/char_rnn_classification_tutorial.html
Placed it here:
https://www.kaggle.com/leshabirukov/char-rnn-class-toolz
(It is not compiling, probably too modern python, but it is working locally at my computer)
Main problem is lower accuracy comparing to original tutorial.

**Comment by Ilykuleshov** [comment]
Fixed in 029c0b8

**Reply by leshabirukov** [reply]
Surprize!  
Now `function` registered twice. Looks like `self.function = args[0]` is enough, `add_module` is not needed. Волшебники хреновы.

**Reply by Ilykuleshov** [reply]
Right, will bump it in the next commit). Да уж)

**Reply by Ilykuleshov** [reply]
052694d seems to work without it, you're right
